CHANGELOG.md

```diff
@@ -1,6 +1,1484 @@
-# Backpack Android Changelog
+# Backpack Android changelog
 
-You can find changelogs here:
+[Unreleased changes](UNRELEASED.md).
 
-- [backpack-android](Backpack/CHANGELOG.md)
-- [backpack-compose](backpack-compose/CHANGELOG.md)
+# 36.0.0
+
+**Breaking:**
+- Removed `buttonStrokeColor` and `buttonStrokeColorPressed` theming properties
+- Replaced `Outline` type to `PrimaryOnDark` for `BpkButton`
+- Removed `BpkButtonWithIcon` base class
+
+**Updated:**
+- `BpkButton` design
+
+**Added:**
+- `PrimaryOnLight` type for `BpkButton`
+- `Large` variant of `BpkButton` and `BpkButtonLink`
+
+# 35.0.0
+
+**Breaking:**
+- Remove deprecated text API
+  - Removed old text styles & token
+  - Removed Relative theme
+
+# 34.0.0
+
+**Breaking:**
+- Upgraded to the new typography system
+  - Deprecated old size scale & weight
+  - Mapped deprecated styles to new consolidated style:
+    - CAPS regular & emphasized, XS emphasized -> Caption
+    - XL heavy -> Heading 3
+    - XXL regular & heavy -> Heading 2
+    - XXXL regular & heavy -> Heading 1
+  - Updated text style + weight property on `BpkText`, `BpkFontSpan` & `getFont` to use a combined `TextStyle` value
+  - Text styles now include line height & letter spacing
+
+# backpack-compose 0.2.0
```

Comment:
At first I kept the old compose releases separately, but then thought it'd be confusing to have a separate file for just the first few releases, so just slotted these in where they were released before